src/js/bail-trends.js

```diff
@@ -56,7 +56,7 @@ const createBailRateTable = () => {
         averages: [
           {
             name: "Avg.",
-            value: 42.59666979
+            value: 38.6
```

Comment:
It occurs to me that these should probably be constants? Since we're using the average in multiple places. Do you mind making that change for all the averages in this file?

Reply:
Done